docs/documentation/release_notes/topics/22_0_0.adoc

```diff
@@ -11,4 +11,11 @@ In a previous release we started to actively log deprecation warnings when the K
 In previous releases, the `export` and `import` commands required a `build` command to be run first.
 Starting with this release, the `export` and `import` commands perform an automatic rebuild of Keycloak if a build time configuration has changed.
 
-See the migration guide for details.
\ No newline at end of file
+See the migration guide for more details.
+
+= Renamed Keycloak Admin client dependencies
+
+After the upgrade to JakartaEE, dependencies for Keycloak Admin clients were renamed to describe the reality more accurately, with consideration for long-term maintainability.
```

Comment:
```suggestion
After the upgrade to JakartaEE, artifacts for Keycloak Admin clients were renamed to describe the reality more accurately, with consideration for long-term maintainability.
```
Maybe we should call it "artifacts"? Not sure about that, but in any case I think dependency is not the right word for what we are distributing.

Reply:
:grin: Yep, agree. Like, in 5 minutes, I've changed the terminology like 5x when I was considering various points of view. But yeah, 'artifacts' might be better.